pages/linux/ncat.md

```diff
@@ -13,3 +13,7 @@
 - Write output of specified file to the specified host on the specified port:
 
 `ncat {{address}} {{port}} < {{path/to/file}}`
+
+- Connect using SSL (for example for HTTPS):
+
+`ncat {{address}} {{port}} < {{path/to/file}}`
```

Comment:
Could we remove the space in between `<` and `{{path/to/file}}` please? It can cause issues on some Linux systems.